queries.sql

```diff
@@ -8,3 +8,35 @@ select name,escape_attempts from animals where weight_kg > 10.5;
 select * from animals where neutered = true;
 select * from animals where name <> "Gabumon";
 select * from animals where weight_kg >= 10.4 and weight_kg <= 17.3;
+select * from animals;
+BEGIN;
+UPDATE ANIMALS SET species = 'unspecified';
+select * from animals;
+ROLLBACK;
+select * from animals;
+
+BEGIN;
+delete from animals;
+select * from animals;
+rollback;
+select * from animals;
+
+BEGIN;
+delete from animals where date_of_birth > '2022-01-01';
+savepoint one;
+update animals set weight_kg =  weight_kg*(-1);
+select * from animals;
+rollback to one;
+select * from animals;
+update animals  set weight_kg = weight_kg*(-1) where weight_kg<0;
+commit;
+
+select count(*) from animals;
+select count(*) from animals where escape_attempts=0;
+select avg(weight_kg) from animals;
+select neutered,sum(escape_attempts) from animals group by neutered order by sum(escape_attempts) desc limit 1;
```

Comment:
**[OPTIONAL]**

- [ ] It would be great if you make your query something like this below 👇 because the query you have has some extra things which are not giving the desired result 🤔

```SQL
  SELECT neutered, MAX(escape_attempts) FROM animals
  GROUP BY neutered;
```

Reply:
SELECT neutered, MAX(escape_attempts) FROM animals
  GROUP BY neutered;
<img width="354" alt="image" src="https://user-images.githubusercontent.com/39811497/228910355-e22b1f4c-c373-4f53-b87f-9f7ad6b74fd5.png">

**Who escapes the most**, neutered or not neutered animals?
The above query not providing the answer of the question. @HAMMAS-SALEEM

Reply:
Hey @sajibarafatsiddiqui I hope you are doing well 😊
When i run your query then it gave me the result which you have in your pull request summary and i am attaching it below ⬇️
![image](https://user-images.githubusercontent.com/93039359/228911429-ac9433f3-5250-4eaa-8242-43a1c32a833b.png)